.github/workflows/early-access.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 env:
   JAVA_RELEASE: '23'
-  JAVA_VERSION: '23.0.1'
+  JAVA_VERSION: 'latest'
```

Comment:
This is fine. However, technically we could remove the env variable 'JAVA_VERSION' as it is no longer needed by the action. Unfortunately, instead of 'JAVA_RELEASE' the value of 'JAVA_VERSION' is used throughout the whole script. For the `oracle-actions/setup-java@v1.4` the only relevant parameter is `release`.